File: baekjoon_all/02000+/boj_2480.cpp
// Solve 2022-06-02
// Update 2023-02-26

#include <iostream>
using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;

    int ans = 0;
    if (a == b && a == c) {
        ans = (a + 10) * 1000;
    }
    else if (a == b || a == c) {
        ans = (a + 10) * 100;
    }
    else if (b == c) {
        ans = (b + 10) * 100;
    }
    else {
        ans = max(a, max(b, c)) * 100;
    }

    cout << ans << '\n';

    return 0;
}

File: baekjoon_all/02000+/boj_2480.py
# Solve 2022-05-16
# Update 2023-02-22 #

a, b, c = map(int, input().split())

if a == b and b == c:
    ans = (a + 10) * 1000
elif a == b or a == c:
    ans = (a + 10) * 100
elif b == c:
    ans = (b + 10) * 100
else:
    ans = max([a, b, c]) * 100

print(ans)
